apache/incubator-openwhisk-deploy-kube
# Testing Improvements

* Support for running :tests:testSystemBasic as part of `helm test`.
    
    * Enable running tests:testSystemBasic for DockerContainerFactory
      configurations (ping test still fails with KubernetesContainerFactory
      because need to fix upstream to drop CAP_NET_ADMIN on user action pod).
    
    * Change error handling strategy in myTask.sh to being mostly
      explicit to get better control and allow retries of commands
      that might be vulnerable to CouchDB's eventual consistiency.
    
    * Remove obsolete script for starting minikube in TravisCI

    * Push openwhisk core git tag forward 2 commits to pick up fix
      for org.apache package rename in tests:testSystemBasic.

    * Separate chart deployment and testing into separate scripts
      because we need to use travis_wait when running helm tests,
      but we want to be able to eagerly see the output of the deploy
      step to enable early aborts of the test run when it gets hung.

Fixes #127.
